earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/globe_cutter_app.py

```diff
@@ -490,11 +527,9 @@ def BuildSearchDb(self, source, polygon):
     self.Status("Querying search poi ids: target=%s" % target)
     poi_list = None
     try:
-      fp = urllib.urlopen(url)
-      http_status_code = fp.getcode()
+      data, http_status_code = self.HttpGet(url)
       if http_status_code == 200:
-        poi_list = fp.read().strip()
-      fp.close()
+        poi_list = data.strip()
     except Exception as e:
       raise Exception("Request failed: cannot connect to server")
```

Comment:
Should we do anything with the exception variable e?  if not, can we remove that variable?